### PR TITLE
update changelog

### PR DESCRIPTION
**Pull request by NoashX**
Update breaking changes for the changelog.

### CODE REVIEW

**Review by annelfitz — APPROVED** [review]
LGTM!

**Comment by NoashX** [comment]
am I okay to merge this @bsvensson?

**Comment by NoashX** [comment]
This PR also updates the Changelog with places @bsvensson.

File: CHANGELOG.md [hunk 1]
```diff
@@ -164,6 +164,19 @@ const someComponents = view.ui.getComponents("top-right");
 
 For`@arcgis/core` ES modules, we improved the console message that warns when there is a version mismatch between the assets used by your build process and the application itself. When working with locally hosted assets, this situation can introduce unexpected issues or errors. The updated warning provides more intuitive information to help you solve the issue faster and easier. It now includes the modules version, assets version and the assets path. These represent the `@arcgis/core` NPM package version used to build the project, as well as the `@arcgis/core` version of the assets and local path to the assets configured in your application. More information is available in the [Build with ES modules](https://developers.arcgis.com/javascript/latest/es-modules/) guide topic.
 
+## Places
+
+We added `esri/rest/places` in `(beta)` to find nearby
```
Comment (annelfitz) [inline CHANGELOG.md:169]: are the parenthesis needed here?
```suggestion
We added `esri/rest/places` in `beta` to find nearby
```

**Pull request merged**